src/view.c

```diff
@@ -652,11 +653,19 @@ update_view(struct view *view)
 			return false;
 		}
 
+		if ((view->pos.offset + view->height + 1) == view->lines)
```

Comment:
autoscroll seems like an interesting feature. I don't use the pager view but I can imagine autoscroll being useful.
I think this commit should be submitted in a separate PR, without the highlighting changes.

There seems to be an off-by-one error. When I type `:!git log` it only scrolls until the last-but-one line; the last line is not visible.

Instead of an option, we could also implement less's F command.

Reply:
@krobelus: I've updated the PR to cover only autoscroll feature and fixed the off by 1 error. I agree that autoscroll is useful, I'm using it all the time.